### ISSUE TITLE
Trip to France

### ISSUE THREAD

**Issue by garrett-sutton** [garrett-sutton/garrett-sutton.github.io#78]
Keeping an issue open that goes through each day so I can write a post about it later

**Comment by garrett-sutton** [comment]
## Wednesday & Thursday (July 17-18, 2024)
- Relatively smooth sailing to fly out of Madison at 11:30 AM
- Landed in Philly about 2:30 EST and had to wait until 8:30 PM to board for Nice
- Got some food and made a few stops in places along the way until we finally got to our actual gate
    - one stop next to the American airlines lounge that was full and we listened to many many people be very upset. Hope to avoid that elitism in my life
- one stressful piece of this was around trying to get Gwen a bassinet for the the red eye to Nice. Had been told by some that I needed to talk to the check-in people in Madison, who told us to talk to our gate agent in Philly. On talking to some others in Philly, they said we had to do it through reservations. And finally we talked to our gate agent only to find out that you need to be sure to book a specific seat (the bulk head seat) to even be eligible. No such language on their website but I guess now we know. Honestly the worst part was just that I knew I had to continue to ask this question that didn't seem to be trending favorably for us...
- Eventually got boarded in Philly and had to sit on the tarmac for 90 mins due to thunderstorms. I was in and out of sleep at that point and don't quite remember take off. Gwen had a rough time falling asleep with me next to her (pre take off) but Grace got her to go to sleep before we even took off. She stayed asleep until an hour left in the flight. Score! Grace and I weren't so lucky. Through various configurations I think we managed to sleep some. Probably better for me than for Grace. I was tired but not feeling exhausted all day. 
- ripped off a bit at currency exchange. They weren't exchanging evenly. 1.3 dollar - 1 euro. But 0.92 euro to 1 dollar. 
- forever wait for rental car. In line and then in a national emerald club specific booth for a while too.
- decently long time in grocery store to find what we needed (without being able to read any French) 
- hour+ drive to La Colmaine (the way the tour is coming up on Saturday). This drive was spectacular.
   - crazy rugged terrain
   - gorgeous rivers/streams running down through them
   - impossible to build roads, villages and buildings all along the way
   - the locals really hated my driving (I was slow on these narrow roads with blind bends... Sorry)
   - some of the most beautiful and continuous terrain I've ever seen
    - the little towns seriously remind you of something from a movie 
- Airbnb is nice. It's a small place in a small ski area, but should work well enough for us. Made sausage and peppers for dinner (which turned out ok) 
- another stress for me that I started to have in the last few days of the waiting for the trip. Language barrier. In just a day, it's been challenging. I feel bad that I know literally no French. Hopefully as we go, I'm a bit less insecure about it and can just roll with it. 
- all along here, definitely had some sour and stressed moments. There's a lot to figure out even with all that we've already figured out. I felt a sense of calm once we got off the main road and headed up to the Airbnb. Things feel a little simpler now that we're out of the city.  
- Gwen has been such a good little girl and Grace has been the absolute best mom to her as well. Happy to have them both with me for this trip

**Comment by garrett-sutton** [comment]
## Wednesday & Thursday (July 17-18, 2024)
- Relatively smooth sailing to fly out of Madison at 11:30 AM
- Landed in Philly about 2:30 EST and had to wait until 8:30 PM to board for Nice
- Got some food and made a few stops in places along the way until we finally got to our actual gate
    - one stop next to the American airlines lounge that was full and we listened to many many people be very upset. Hope to avoid that elitism in my life
- one stressful piece of this was around trying to get Gwen a bassinet for the the red eye to Nice. Had been told by some that I needed to talk to the check-in people in Madison, who told us to talk to our gate agent in Philly. On talking to some others in Philly, they said we had to do it through reservations. And finally we talked to our gate agent only to find out that you need to be sure to book a specific seat (the bulk head seat) to even be eligible. No such language on their website but I guess now we know. Honestly the worst part was just that I knew I had to continue to ask this question that didn't seem to be trending favorably for us...
- Eventually got boarded in Philly and had to sit on the tarmac for 90 mins due to thunderstorms. I was in and out of sleep at that point and don't quite remember take off. Gwen had a rough time falling asleep with me next to her (pre take off) but Grace got her to go to sleep before we even took off. She stayed asleep until an hour left in the flight. Score! Grace and I weren't so lucky. Through various configurations I think we managed to sleep some. Probably better for me than for Grace. I was tired but not feeling exhausted all day. 
- ripped off a bit at currency exchange. They weren't exchanging evenly. 1.3 dollar - 1 euro. But 0.92 euro to 1 dollar. 
- forever wait for rental car. In line and then in a national emerald club specific booth for a while too.
- decently long time in grocery store to find what we needed (without being able to read any French) 
- hour+ drive to La Colmaine (the way the tour is coming up on Saturday). This drive was spectacular.
   - crazy rugged terrain
   - gorgeous rivers/streams running down through them
   - impossible to build roads, villages and buildings all along the way
   - the locals really hated my driving (I was slow on these narrow roads with blind bends... Sorry)
   - some of the most beautiful and continuous terrain I've ever seen
    - the little towns seriously remind you of something from a movie 
- Airbnb is nice. It's a small place in a small ski area, but should work well enough for us. Made sausage and peppers for dinner (which turned out ok) 
- another stress for me that I started to have in the last few days of the waiting for the trip. Language barrier. In just a day, it's been challenging. I feel bad that I know literally no French. Hopefully as we go, I'm a bit less insecure about it and can just roll with it. 
- all along here, definitely had some sour and stressed moments. There's a lot to figure out even with all that we've already figured out. I felt a sense of calm once we got off the main road and headed up to the Airbnb. Things feel a little simpler now that we're out of the city.  
- Gwen has been such a good little girl and Grace has been the absolute best mom to her as well. Happy to have them both with me for this trip

**Comment by garrett-sutton** [comment]
## Friday 7/19
- Managed to sleep in a little bit. Gwen was up in the middle of the night for a while but after she went to back to sleep, she slept until after 7!
- I went for a quick run. Down the mountain to St Dalmas and back up. 750ft in 2 miles is something. Stunning view all around though. It was beautiful to see both ways. Down into the valley and back up. Not too shook after that but my heart rate was flying. The running around here will be difficult. It's either up or down.
- I made some horrid eggs (I partially blame the classically bad Airbnb cookware)
- We walked down to the La Colmaine village but there wasn't much going for us. Passed by the few restaurants that all looked closer and went to the lookout. Decided it was best to get on the road asap for Isola.
- Drove to Isola with limited confidence about the parking situation. At a few points, I figured it just wasn't going to happen. But eventually there was an open lot where the police even helped us back in. Gwen ate, Grace pumped and we packed ourselves up for the mile walk and a few hours of waiting (Gwen in the carrier. Thank goodness we didn't take the stroller - it would have been too much).
- Little over a mile walk to Isola Village and we tried to find some food/coffee. It was super busy and sort of stressful with not being able to read most of the signs (and not having service to Google translate). We eventually just waited in line at the grocery store and got a few things. The Village itself was so picturesque alps village though. Cobbled streets, a couples squares, shuttered windows all over. 
- went down to a park from the base of the final climb to hang out for a little bit, eat, and let Gwen move some. 
- Caravan came through. This was super super cool. I thought it was just some tour vehicles that played music and tried to hype people up. But it's so much more. There are tour vehicles and music, but all of the sponsors also have vehicles and they are all throwing out souvenir stuff. I managed to snag a cycling hat, a green jersey bucket hat, a polka dot bucket hat, a reusable bag, dish wash detergent, and coffee. It was wild to see all of the people collecting these huge hauls of stuff. Fun though. We'll hopefully be more prepped tomorrow for other stuff. It was really cool to see how excited everyone was and the total party/community atmosphere for it. 
- after the caravan we found a good spot to see the riders come through and there also happened to be another screen that we could watch (in addition to the one in the village square). This largely became a waiting game in a really pretty place, watching some of what was going on, taking in the people and enjoying the absolutely beautiful day. It was overcast and about 75 all day. That's perfect for me. 
- The riders eventually came through. We largely watched the first 2 groups (G1 had Matteo Jorgenson (USA) and G2 had Pogi, Jonas and Remco). After they all made it through, we went back up to the square and found a seat to watch the rest on the big screen. Super exciting finish. Tadej made up like 3 minutes over the last 5 miles to catch and beat Matteo (Matteo did finish second). 
- After they finished, we started making our way to the car and heading back for our Airbnb. Took a pit stop at the church in St Dalmas for Grace to see (I ran by in the morning). Unfortunately it wasn't open. 
- Got back to the Airbnb, and we sort of chaotically for ourselves through the rest of the night. Grace cooked chicken and orzo and fed Gwen, I showered and have Gwen done real good. Grace bathed her while I cleaner up. Eventually we all had it together and hit the hay

**Comment by garrett-sutton** [comment]
## Saturday 7/20
- up to run a little bit. Feel tired from I think the combo of travel, altitude, walking yesterday and running yesterday
- out to the Zipline at Colmiane. Gwen and I rode the chairlift up and down. Grace did the Zipline. Super cool all around. Beautiful mountain terrain. It looked higher (out maybe we were just higher) than what we saw yesterday. More above treeline jagged peaks and even some snow.
- back to Airbnb for lunch, some packing and a quick nap before heading down for the caravan
- while waiting for the caravan, met a few women from the US. Talked about Epic, their background (she was from Connecticut but is now in Chicago)
- saw the Caravan. Sadly no yellow cycling hats were thrown. But got a haul of a bunch of other stuff. Organia (orange soda) have be a whole flat of cans. And the polka dot sponsor just hanged out hundreds of shirts. I snagged 2. Got a bunch of other little stuff too. 
- after caravan at the top of the col, we went down the mountain a little bit to get on the climb more and hopefully see more. Tons and tons of people. Super cool. Hung out in the woods for a bit while we waited. It was all a big picnic basically. People just everywhere, hanging out and waiting.
- when the riders arrived, it was electric. Could definitely see how the crowds widen and narrow with the police escorts and the riders. Stayed in this spot until everyone was through. While didn't feel terrible long apart. Gave Neilson and Sean Quinn warm USA chants. The groupeto was wild too. They actually took up the whole road which left us scrambling to the side to get out of the way. Unfortunately didn't see Cav but did see Bini.
- walked back up to the top and watched the finish on the big screen. Another win for Tadej, exclamation mark on this tour. Many many others did too. The picnic vibe just moved up the hill to the screen.
- went back to Airbnb, packed up quickly, ate some of our leftovers for dinner and hit the road for Nice
- relatively long (the roads are just slow) but gorgeous ride to nice. Saw people riding bikes (seemingly like they'd been at the stage) and noticeable change in environment. The mountains are like all fir trees but you can really notice the vegetation look a lot more tropical (ish?) as you come down into Nice. 
- got into our parking garage easy and stumbled upon the hotel. Pushing stroller and bags a bit painful on the city sidewalks but we made it. Notice they don't have ADA ramps like we do in USA
- super Small hotel room. Unable to fit stroller into their elevator... Pleasant. Only had to tote it up 2 flights of stairs. Also no baby accommodations. I must not have communicated with them that I needed a crib and only saw that they had them available on the listing. We managed a makeshift crib for Gwen. She had a really rough time falling back to sleep unfortunately.

**Comment by garrett-sutton** [comment]
## Sunday 7/21
- Started the day by just getting to mass at Notre Dame Basilica of the Assumption. Beautiful church, but didn't understand most of the language, but can follow with what is going on. Individual chairs and hard kneelers
- stopped for coffee and bagels on the way back to the hotel. Many many powdered eggs and a small small Americano were all good for us
- wasted money on tram tickets to get to the main area of Nice. It wasn't that far and then the train didn't even go to our stop. It had to stop because of the Tour finish. 
- went over to Old Nice. Visited the garden on the hill - Gwen got to play in the park there (artificial grass was nice). Although overcast, it was beautiful. Came down from there to the 4km to go banner to wait for the caravan 
- snagged the covetted yellow hat in the caravan! Also met a friendly French lady 
- went into the depths of old nice to discover the Church of St Jacques. Really old and beautiful. So much detail. Also wanted to go to the cathedral but it was closed for the afternoon so we got pizza on the square outside of it and hung around for a bit. These squares seem to be everywhere and it's an interesting tourist attraction/community gathering space. Cool how it fits into the whole "city plan"
- then to the beach for a little while. Gwen loved the rocks - a bit too much - put some in her mouth... Water was pretty warm. 
- then back over to the race. We made our way to around the 200 to go signs, Grace and I shared a beer and then went into the finishing area. Couldn't see anything in the finishing area and Gwen needed cleaned up&to eat so left there to take care of business. Back in the bar, Gwen got cleaned and ate. Then we posted up on the course just down from that bar to watch the big guns come through (250 to go). Saw Matteo, Remco, Jonas and Tadej (among many others). The crowd was electric. Super cool to see how quick they are ripping through there. Really cool to see so many people from so many places to be ready to cheer every 2 minutes for like 4 hours. 
- moved quickly after they passed us to get back to the hotel for Gwen to finish up her night (our team tickets were basically no good because it was closed and then we actually had to walk a long way and would have only been on it for 1 stop.). After she got a bath and was eating, I ran out to get some food. 
- Grace and I enjoyed pizza, tiramisu and waters on the hotel balcony and talked about the unique and authentic nature of Nice. I think it's the most unique town I've ever been in. Just because it is all so the way it is. It's not a show. It's just always been that way seemingly (I'm assuming this because everything seems so old). 
- Feel privileged to have seen today's finish in Nice, seeing all that we did, and have Grace and Gwen there all the way. Gwen had a few moments but all in all was (and continues to be) a phenomenal baby. She's been very flexible and does a good job being in the carrier for long periods in time. Grace is a rock for the 2 of us. Keeping a cool head while Gwen is loosing it or I'm stressing about something else.

**Comment by garrett-sutton** [comment]
## Monday 7/21
- up and at em to run before hitting the road from Nice. Nice felt different in the early morning. I feel like I only liked it during the hustle bustle of the day. Nice to run along the beach though
- got things situationed and grabbed breakfast from a little cafe. Good all around. Americano here is interesting. A lot less volume compared to what I'm used to. But was good.
- getting ice for breast milk was a dilemma. Restaurants and coffee shops were unable to help because of bad ice machines and then I got laughed out of a hotel for even asking. In another hotel, it felt like we were transported back to WI because they didn't ask any questions and told us "here ya go" (even with a non to go cup). 
- then the drive started. Wanted to make a few stops along the way at a national park and a Notre Dame chapel. The national park rt took us up over a big pass into the middle-ish of nowhere. The lookout point was crazy. Very very deep and tons of exposed rock. 
- getting to the chapel was another story. We spent close to an hour on a dirt road through actual nowhere (no houses no nothing). This was scary. Lots of prayer and a rosary during it. We got back to hard road and then immediately were supposed to go down a dirt road and I was just done. The stress of a rental in a foreign country in an area with no service when it's 90 degrees out with a 9 month old was just a lot. Got back on hard road and still really small roads. Came across an area where lavender was a big thing. Some fields and farmers harvesting it. Very pretty. 
- eventually got onto a highway and made decent time towards Briancon. Stopped for dinner and ended up with Pizza again. Neither Grace or I were super impressed. But we'd also had a lot of pizza in 48 hrs. Language barrier here and the odd realization that many restaurants don't open until 7 for dinner. 
- then headed the last 30 min to Briancon. As we were making our way to our Airbnb, saw a guy running with blood on his face. Soon after saw cops around. And eventually we saw the guy and cops seemingly looking for him again. A bit unsettling, but we got unloaded safely and got to bed soon after.
- tough night for Gwen falling asleep. Transitions are hard for everyone.

**Comment by garrett-sutton** [comment]
## Tuesday 7/23
- up and tried to find coffee/breakfast. A bit weird feeling but many places seemingly only have pastries in the morning and their kitchens open later on. Got coffee and croissant, walked to grocery store to stock up.
- I rode the Granon while Grace and Gwen stayed back. Rental went super smooth. Rode a full carbon, Scott bike, size 54, Shimano 105. Ride up was hard. Stopped after 6 km up to get a snack while not riding and drink some. Able to make it the rest of the way with not a lot of trouble. Just slow. I couldnt help but think a lot about the 2022 stage here and how fast they must have been going and even how fast the last guy was going compared to me (and I didn't have the Telegraph and Galibier in my legs too). It was fun to get up there though. Absolutely beautiful and well worth it. Grabbed a sparkling water and coke and the top and sat in a beach chair for a few minutes before heading back down. Also noted some pretty wildflowers on the way. Way down was smooth sailing and just making sure to stay well in control. 
- Found Grace and Gwen at a nearby park that had a good view of the fortified Briancon. Espresso and hung out for a little while to get Gwen moving some.
-Went out for a beer/cocktail before dinner. Galibier Pale Ale was really good. 
- Grace made a really good dinner (thankfully no pizza) of chicken stir fry. Gwen hit the hay. Grace and I planned the hike for tomorrow, played some cards (I lost big time) before getting to bed.

**Comment by garrett-sutton** [comment]
# Wednesday July 24
- up and out the door by 8 for a hike connecting the forts of Briancon. 0 service so ended up remember guessing where we were supposed to go with the all trails location feature. It worked out pretty well. Gwen was awesome. The sun wasn't too bad until the very end and even then it wasn't terrible. Just started to feel warm at the end of the 7ish miles. Really really pretty hike. Could get above the fortified city of Briancon and see up the valley that would take you to Italy. Really cool we could do all of this with never even moving our car. We went right from Airbnb. 
- ate lunch at the Airbnb and then I headed out for a run. Found a more commercial part of town that was pretty flat and ran some loops around there. Also touched the Col d' Izolard but only a bit. Happy I didn't go much more because coming down into traffic was a bit sketchy (which is weird considering I didn't have a designated lane going up and it felt less sketchy)
- went to get gelato and some groceries before dinner. Funny that almost all of the good we've gotten here is actually Italian. So much that I can't say I really know what French food is.
- Gwen is terrified of the robot vacuum. Even when she turns it on herself. We shut it off a few times and she'd attack it and turn it back on. Poor little girl
- steaks for dinner in the pan came out really good. 
- after Gwen went to bed, some reading and more of me loosing at cards

**Comment by garrett-sutton** [comment]
## Thursday July 25
- up for an hour run. Very easy. Very pleasant. Cool and calm. 
- Grace ran too, then we got ourselves together to go to the ski area.
- stopped and sat down for coffee on the way. Language barrier again bit me as I continue to speak English when the woman couldn't get what I was saying. All ended up alright. Coffee was good too 
- half mile walk through town to the ski area. Weird language encounter there. A woman started giving us instructions in French and I said sorry but do you speak English and she answered that she did but we are in France and then proceeded to begrudgingly give me the directions in English. Sort of annoying. Sort of whatever. 
- hiked around at the top of the lift. Beautiful views all around. Some semi technical stuff too. Walked to the Notre Dame des nieges chapel that overlooks the city of Briancon too. Getting to be around 1 pm so we headed down and got back to the Airbnb 
- quick eat lunch and headed back out the door for the Col du Galibier (driving). Absolutely stunning drive of the serre Chevalier valley and then the Galibier on top of that was something extra. Incredible winding roads up to the Col for 360 degrees views. Super cool to see how the tour would have come through here in 2022 and now in 2024. This may be the coolest thing I think we've seen in this trip. The Granon was awesome to ride up but the view at the top of the Galibier and how you can see the windingness roads is so so cool.
- As she has been, Gwen was really good today. Played nicely in the Airbnb, fended off the robot vacuum from a few more self-inflicted attacks. She did really well in the car and while Grace was carrying her. Happy happy baby. Thank God!

**Comment by garrett-sutton** [comment]
## Friday 7/26
- largely a travel day
- up for a quick run before coffee (and croissant) and hitting the road in Briancon
- drive from Briancon to Grenoble and especially to a town that I think was called La Gavre was stunning. Repeat route to the Galibier that we took yesterday and then beyond down the other side of the col de lauduret. 
- made it to Grenoble, figured out the car return situation and found a restaurant inside the train station. Different car return than usual. Also the first time paying for the bathroom. 
- stress about the train and bag and timing situations (making sure we can actually get off etc)
- Gwen was really busy in the train after only sleeping for an hour of the 3 hour trip 
- arrived in Paris and managed to navigate the metro with all of our bags in tow. Not all stops and areas have accessible things. Something I take for granted in the US. everything else was otherwise. But we managed to get to the hotel in good spirits. 
- after settling, I ordered and went to pick up some food. We watched the opening ceremony on the TV for the rest of the night. Managed to rig up a curtain to separate Gwen's pack and play from the rest of the room

**Comment by garrett-sutton** [comment]
## Saturday 7/27
- rainy all day long
- rained on my run. I got a bit lost but managed to figure it out.
- left with plan of cafe and Miraculous Medal chapel, but immediately diverted to grocery store to get umbrellas that were very useful all day
- then cafe, Miraculous Medal chapel. This was really cool. Powerful to see an apparition site and pray through the end of mass and have the medals that we bought blessed (although hard to understand really what was going on with it being in French). Gwen ate here and promptly fell asleep. Really interesting that the foundations of the Dogma of the immaculate conception come from Saint Catharine Labours apparition. Our own blessing in this journey. Grace had to pee super bad and the public toilet was not working. This really nice woman came over and told us that there was a bathroom in a store close by that was easy and much nicer. Store worked out super well. But then when we went to the Chapel, that lady was there again and said hi. This store was helpful again in the evening when I was the one who really had to pee.
- Walked over to the outdoor part of the Louvre. Stopped at a coffee shop along the way. Saw the pyramid and other garden pieces in the middle of the Louvre. Made our way to another coffee shop for some respite from the rain. Gwen crawled around a bit too. 
- walked over to the Champs Elysses and the Condors obelisk. Cool to see the ard de triumph in the distance and the cobbled road. Area riddled with people and Olympic detours
- then went to the time trial course. Watched some of the women come through.
- found another church (Basilica of St Clotidle) for Gwen to eat and get changed in after struggling to find a cafe that was open enough for us to sit and wouldn't be too distracting for Gwen to eat. 
- found a dinner spot right along the course again for the men's TT race and got to see everyone go past 2 times (out and back). 
- hustled on back to the hotel after dinner to get Gwen bathed, fed and to bed. She only ever slept for maybe an hour today but was really a great baby the whole time. She did so so well to be out from 9 am to 7 pm. Certainly a happy happy baby. 
- fun day. Paris is a really pretty city (lots of green), even when it rained basically all day long. The old buildings and how well they are kept everywhere is just mind boggling to me. There is so much detail everywhere you look. This also feels a touch more homey because it looks more like the US. Small shops for big name brands all around and there's a lot of English speaking you can overhear in the streets too. Still lots of language barrier, but you don't feel so along with it.

**Comment by garrett-sutton** [comment]
## Sunday 7/28
- Another day in Paris
- up to run before mass. Went down around Notre Dame, the Pantheon and the gardens of Luxembourg. This town has something interesting/notable around what seems like every corner.
- Mass at Saint Sulpice. This church was incredible. Huge, with huge statues and so much detail everywhere. Marian Chapel in the rear was stunning. We originally sat in the big part of the church only to find that the mass was happening behind the main altar. Grace fed Gwen at the beginning and she slept for all of mass. Mass was in French, so somewhat difficult to follow at times and I admittedly fell asleep during the homily. They used the Latin for various pieces that were familiar though and you can obviously follow along with where we are in the mass. Again, just a stunning stunning church. Even though it's old and in the process of being restored, it's still stunning.
- after mass, went to a cafe for breakfast. They push orange juice, hot beverage and pastry for breakfast. It's good but needed more food. Got it for the most part with the egg and crepes we ordered too.
- walked down to Notre Dame and Saint Chapel. Made a loop. Beauty all around. 
   - Also used the public restrooms below Notre Dame. 2.50 euro per person... Finding bathrooms has been somewhat difficult while in Paris. This situation was more weird because the guys working seemingly didn't speak English and their turn style wasn't working? So they had to use their card machine to actually let people in (rather than the turn style terminal). At a point, it also seemed like they were going to charge us for Gwen too. But they did have a changing table and that was a win.
- lingered around Notre Dame area a bit and then hit the metro to get to the Eiffel Tower. Smooth sailing. Got a good spot to see and take some pictures before the volleyball games. Eiffel Tower was a lot bigger than I ever really expected it to be. Much cooler than I would have expected. Lots of people all around.
- Volleyball games were a lot of fun. No team USA for us. We got to see Netherlands vs Lithuania women, Brazil vs Egypt women and Canada vs Czech Republic men. All ended in 2 sets. But a really fun atmosphere. Seeing the Eiffel Tower and the game all at once was real cool. And then the whole stadium area also had grass and other stuff for people to interact with, notably misters that Grace said Gwen loved. Gwen did pretty good too. A bit fussy at points but a few breaks from the action and good snacks kept her going good. 
- metro on the way home, interesting train car that you had to open the door. Got "Broadway Cafe" takeout on the way. This place had a full on taxi inside of it. Waiters wearing Yankees hats. Interesting how American culture and especially NY permeates

**Comment by garrett-sutton** [comment]
## Monday July 29
- up and at em to run and get out the door 
- series of interesting metro navigations with stairs and stalls not big enough for the stroller but we made it to the train station with plenty of time. We've been hit and miss with some French people but getting to the station this morning was filled with very good souls. Helping us carry the stroller up the stairs and talking to someone to get us out of the metro station (stroller wouldn't fit).
- got on the train. A bit of chaos finding places to put our luggage. Gwen played in her seat and then with some random stranger kids for a bit. She did pretty good and eventually fell asleep and stayed asleep for a while 
- got a car in Caen. Found lunch in this cool alleyway with a bunch of restaurants. 
- looked at the Church of St. Peter for a bit. Stunning church. Like I've said before, there's just so much detail. It's nearly impossible to encapsulate in pictures. 
- stopped in at the caen castle only to find the museum we wanted wasn't there. Good thing we went bailed in that museum because the next one needed all the time we could get
- memorial de Caen. Really cool museum with a lot of info around soldiers, civilians, allies, axis, timeline and devastation
- also talked briefly with a guy who asked where I was from. I said US and he responded " you know you guys are the reason we have a free Europe today, otherwise we'd all be speaking German." Moving words to just try to understand the gravity of the US's position in the world
- left there to go to our hotel. I picked a hotel that I knew would have parking but it was a ways outside of town. This means only a single restaurant nearby and they unfortunately don't do take out (which also seemed ridiculous). With no one on Uber eats and me not wanted to go back out for another hour of driving, we ended up settling on hotel vending machine. This food situation is highlighted in full tonight but has been a problem all trip. We need to do takeout because Gwen needs to go to bed and the restaurants don't seem to be open until 7. Seemingly restaurants are a gathering place here. Not a get it and get out deal sort like what we do in the USA.

**Comment by garrett-sutton** [comment]
## Tuesday 7/30
- hotel breakfast and on the road
- visited airborne museum, drove by Utah Beach, saw the dick winters memorial, walked a loop at Pointe Du Hoc and drove through Omaha beach. 
   - Felt like a rushed visit given there is just so much to see. But in what we did see, I found myself trying to put faces to names. I'll hopefully forever be able to see these sites in the future when reading or watching things about DDay. 
    - feels like it's presented in the context of the whole war which is helpful and interesting to think of. Found myself looking out into the sea and imagination the of 10,000 boats, 15000 planes and 150000 soldiers. It's lll 
- drove back to Caen and got yelled at by a gas station attendant 
   - I was just trying to fill up and they didn't have a card scanner. In the process of me trying to figure out what I actually needed, the guy seemingly got angry and started forcefully telling me that I needed to speak French. 
- train back to Paris. Gwen was in a needy state
- then metro/subway to the airport. Felt bad for Gwen. Super hot on the train and long ride to get out there. And it was getting late. She was honestly a champ through it all.
- getting to the hotel was a dog eat dog world with our hotel shuttle. Luckily we only have to do one extra wait 
- TODO: more on my feeling of Normandy beaches

**Comment by garrett-sutton** [comment]
## Wednesday 7/31
- run in the morning
- breakfast at the hotel 
- flight delayed by 3 hours but went to the airport on 11:30 shuttle anyway just to be in one place
- made it through security easily. Pretty smooth process. Notably didn't need to take off shoes and they had a family line. 
- Paris airport has PS5s
- Finally head out 2.5 hours late. Makes the layover in Chicago way tight due to getting through customs, baggage and would have also needed to go through security
- American rebooked us for 8/1 at 5 pm, a hotel that may have had transport and $12 each for a meal voucher. Way too kind of them. We got a one way rental instead and just headed home. After ~24 hours of limited/no sleep (but only 16 hours on a clock), we made it home.
- Gwen was notably a champ through it all. She slept well on the flight, was a great sport as we were figuring out our logistics and slept most of the way home too
- the long flight was a bit nerve-racking to me given it being during the day against the Jetstream, but it went really well. Maybe even better than the one going over. Watched the first 3 episodes of masters of the air. Good show
